azurelinuxagent/agent.py

```diff
@@ -208,28 +209,30 @@ def collect_logs(self, is_full_mode):
         else:
             logger.info("Running log collector mode normal")
 
+        LogCollector.initialize_telemetry()
```

Comment:
Previously this was happening in LogCollector init. 
We don't initialize the LogCollector until after cgroup checks, which resulted in all cgroup check events having initialized common properties (see PR description)